web/client-api/src/main/java/io/deephaven/web/client/api/subscription/ViewportData.java

```diff
@@ -86,6 +86,10 @@ public ViewportData(RangeSet includedRows, Object[] dataColumns, JsArray<Column>
             if (c.getFormatStringColumnIndex() != null) {
                 data[c.getFormatStringColumnIndex()] = dataColumns[c.getFormatStringColumnIndex()];
             }
+            // TODO: Figure out why I don't need this section here
+            // if(c.getFormatDataBarColumnIndices() != null) {
+            // c.getFormatDataBarColumnIndices().values().forEach(idx -> data[idx] = dataColumns[idx]);
+            // }
```

Comment:
You probably do still need this.
Make a table with like 100 columns, and add databar formatting to the first column (so you can only see the first few columns). We still want to have these columns returned.